services/core/shared/core/queue.js

```diff
@@ -41,10 +41,8 @@ const initializeQueue = (queueName = 'defaultQueue', queueJob, options = config.
 		logger.debug(`${queueName} Job error`, err);
 	});
 	queue.on('failed', (job, err) => {
-		logger.debug(`${queueName} Job failed`, err);
-		const { data } = job;
-		queue.add(queueName, data, data.options);
-		job.remove();
+		logger.warn(`${queueName} Job failed`, err.message);
+		logger.warn(`${queueName} Job failed`, err.stack);
```

Comment:
Removing these lines will cause the queue to not retry the failed jobs automatically. Hence, we need to explicitly specify extra configuration every time we add a new job to the queue with `queue.add()`. This calls for additional overhead.
**Refer**: https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#queueadd

Reply:
It is already taken care of, the default config sets `attempts` to `5`.